packages/clerk-js/src/ui/styledSystem/common.ts

```diff
@@ -28,7 +28,28 @@ const textVariants = (t: InternalTheme) => {
     lineHeight: t.lineHeights.$small,
   } as const;
 
+  const sectionTitle = {
+    ...base,
+    fontWeight: t.fontWeights.$medium,
+    fontSize: t.fontSizes.$md,
+    lineHeight: t.lineHeights.$normal,
+  } as const;
+
+  const title = {
+    ...base,
+    fontWeight: t.fontWeights.$bold,
+    fontSize: t.fontSizes.$md,
+    lineHeight: t.lineHeights.$small,
+  } as const;
+
   const subtitle = {
+    ...base,
+    fontWeight: t.fontWeights.$normal,
+    fontSize: t.fontSizes.$md,
+    lineHeight: t.lineHeights.$small,
+  } as const;
+
+  const label = {
```

Comment:
This is not what we want to do. We need to be aligned with the Figma design system. If something is off, then we need to fix the designs.